tests/sentry/sentry_metrics/test_parallel_indexer.py

```diff
@@ -28,7 +28,7 @@
 }
 
 
-def test_basic(request):
+def testass_basic(request):
```

Comment:
lol

Reply:
I am testing the reviewers skills and see whether they are able to catch these 🤣